.eslintrc.yaml

```diff
@@ -0,0 +1,145 @@
+parser: "@typescript-eslint/parser"
```

Comment:
I would suggest using a `.js` file for the config as it can be linted also.

Reply:
But it only needs linting if it's js, though... Wouldn't that be solving a problem that this creates? 🤷🏻‍♂️

Reply:
I get your point but this is to benefit from the "style" rules of the linting config (indentation, comma-dangle, max-len, etc.)